### PR TITLE
NFDIV-1413: Configure cron job for bulk case migrations

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

File: k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      environment:
+        NOTIFY_TEMPLATE_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.perftest.platform.hmcts.net/
+        NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.perftest.platform.hmcts.net/
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.perftest.platform.hmcts.net/applicant2
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.perftest.platform.hmcts.net/applicant2
+        S2S_URL: http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal
+        FEE_API_URL: http://fees-register-api-aat.service.core-compute-perftest.internal
+        IDAM_API_REDIRECT_URL: https://div-pfe-perftest.service.core-compute-perftest.internal/authenticated
+        IDAM_API_BASEURL: "https://idam-api.perftest.platform.hmcts.net"
+        CASE_DATA_STORE_BASEURL: "http://ccd-data-store-api-perftest.service.core-compute-perftest.internal"
```
Comment (nitinprabhu) [inline k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml:19]: don't think we need those urls as they will be replaced based on environment

Comment (tomxelliott) [reply k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml:19]: ok cool, will remove 👍

**Review by nitinprabhu — APPROVED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/aat-00.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: nfdiv-cron-migrate-bulk-cases
+spec:
+  releaseName: nfdiv-cron-migrate-bulk-cases
+  values:
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: aat
+      schedule: 0/10 20-23,0-7 * * *
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/aat-00.yaml:13]: is this schedule supposed to be in the global section?

File: k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  releaseName: nfdiv-cron-migrate-bulk-cases
+  values:
+    job:
+      suspend: true
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: perftest
+      schedule: 0 0 * * *
```
Comment (timja) [inline k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml:15]: according to docs schedule is not a global value
https://github.com/hmcts/chart-job#configuration

**Review by timja — CHANGES_REQUESTED** [review]

File: k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml (hunk 3 above)
Comment (tomxelliott) [reply k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml:15]: @timja Interesting, we have it configured in this line: https://github.com/hmcts/cnp-flux-config/blob/7a691baf54b0824bb15a71eeca07148f61989b6e/k8s/namespaces/nfdiv/nfdiv-cron-migrate-cases/perftest-00.yaml#L29

**Review by nitinprabhu — APPROVED** [review]

**Review by timja — CHANGES_REQUESTED** [review]
not fixed in aat

**Review by nitinprabhu — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**